pkg/controllers/configobservation/console/observe_consoleurl.go

```diff
@@ -20,8 +23,23 @@ func ObserveConsoleURL(genericlisters configobserver.Listers, recorder events.Re
 	listers := genericlisters.(configobservation.Listers)
 	errs := []error{}
 
+	clusterVersionConfig, err := listers.ClusterVersionLister.Get("cluster")
+	if err != nil {
+		return existingConfig, append(errs, err)
+	}
+
+	isConsoleCapabilityEnabled := false
+	for _, capability := range clusterVersionConfig.Status.Capabilities.EnabledCapabilities {
+		if capability == configv1.ClusterVersionCapabilityConsole {
+			isConsoleCapabilityEnabled = true
```

Comment:
nit: you could `break` here, but :shrug:, iterating through the remainder of the list isn't expensive.

Reply:
there's a `break` in the next line now, so this thread can be marked resolved.